spinta/manifests/internal_sql/components.py

```diff
@@ -0,0 +1,79 @@
+from typing import Optional, Dict, List, Final, Literal
+
+from spinta.manifests.components import Manifest
+import sqlalchemy as sa
+
+
+class InternalSQLManifest(Manifest):
+    type = 'internal'
+    path: Optional[str] = None
+
+    @staticmethod
+    def detect_from_path(path: str) -> bool:
+        try:
+            url = sa.engine.make_url(path)
+            url.get_dialect()
+            engine = sa.create_engine(url)
+            with engine.connect() as conn:
+                meta = sa.MetaData(conn)
+                meta.reflect()
+                tables = meta.tables
+                return list(tables.keys()) == ["_manifest"]
+        except:
```

Comment:
Never leave bare `except:` statements, because they catch everything, including syntax errors. Be specific on what errors are expected and only add expected classes.

https://python.land/how-not-to-handle-exceptions#The_worst_way_to_do_it